octree/include/pcl/octree/impl/octree_iterator.hpp

```diff
@@ -317,7 +317,8 @@ namespace pcl
       OctreeBreadthFirstIterator<OctreeT>::reset ();
 
       // Set the current_state_ to the right depth
-      while( this->current_state_ && ( this->getCurrentOctreeDepth () != depth_ ) ) {
+      while ( this->current_state_ && ( this->getCurrentOctreeDepth () != depth_ ) )
```

Comment:
I'm afraid we still have style problems here :D 
There shouldn't be spaces inside brackets, i.e.

```cpp
while (this->current_state_ && (this->getCurrentOctreeDepth () != depth_))
```